lib/systems/include/BaseController.h

```diff
@@ -1,16 +1,25 @@
 #ifndef BASECONTROLLER
 #define BASECONTROLLER
 #include "SharedDataTypes.h"
+
+/// @brief defines namespace for definition of a drivetrain command with no torque for clearer code in the Muxer
```

Comment:
this namespace is for more than that: its for all base parameters that can constructed for arguments to any base controllers as a place to store random constant structs